#1: don't fail if misc/dialog is installed (conflicting dialog.h)
#2: avoid -Werror

Index: cmake/platforms/unix.cmake
--- cmake/platforms/unix.cmake.orig
+++ cmake/platforms/unix.cmake
@@ -102,7 +102,7 @@ else()
   set(NOT_X_WINDOWS ON)
 endif()
 
-include_directories(${CMAKE_SOURCE_DIR}/charset ${GTK_INCLUDE_DIRS} ${X11_INCLUDE_DIR})
+include_directories(${CMAKE_SOURCE_DIR}/charset ${CMAKE_SOURCE_DIR} ${GTK_INCLUDE_DIRS} ${X11_INCLUDE_DIR})
 link_directories(${GTK_LIBRARY_DIRS})
 
 function(add_optional_system_lib library testfn)
@@ -211,7 +211,7 @@ endif()
 
 if(STRICT AND (CMAKE_C_COMPILER_ID MATCHES "GNU" OR
                CMAKE_C_COMPILER_ID MATCHES "Clang"))
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Werror -Wpointer-arith -Wvla")
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wpointer-arith -Wvla")
 endif()
 
 function(installed_program target)
